### PR TITLE
Coerce deal value to string before update

### PR DESCRIPTION
**Pull request by michalsapka**

### CODE REVIEW

File: lib/DealsService.php [hunk 1]
```diff
@@ -60,6 +60,7 @@ public function create(array $deal)
     $attributes = array_intersect_key($deal, array_flip(self::$keysToPersist));
     if (isset($attributes['custom_fields']) && empty($attributes['custom_fields'])) unset($attributes['custom_fields']);
  
+    $attributes["value"] = Coercion::toFloatValue($attributes['value']);
```
Comment (pawelgocek) [inline lib/DealsService.php]: Did you mean toStringValue here?

File: README.md [hunk 2]
```diff
@@ -237,6 +237,15 @@ Actions:
 * Update a deal - `client->deals->update`
 * Delete a deal - `client->deals->destroy`
 
+**Note about deal value**
+
+You can use either a string or numerical deal value when modifying a deal.
+```php
+$deal['value'] = 10;
+$deal['value'] = 10.10;
```
Comment (pawelgocek) [inline README.md]: So in PHP there are no precision loss problems?

Comment (michalsapka) [inline README.md]: More like "there is no alternative that I know of"

Comment (pawelgocek) [inline README.md]: Shouldn't we recommend strings anyway?

Comment (michalsapka) [inline README.md]: We have decided (with @grzegorz-mysliwiec) that we want to use the same type that the client returns.

File: lib/DealsService.php (hunk 1 above)
Comment (michalsapka) [inline lib/DealsService.php]: Fixing in progress.

File: README.md (hunk 2 above)
Comment (pawelgocek) [inline README.md]: Ok, got it, so there is no decimal type in python?

Comment (michalsapka) [inline README.md]: None that I could find.

**Pull request merged**